solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -865,9 +894,15 @@ pub struct RemoveUserAuthorityDelegate<'info> {
         seeds = [&user.key().to_bytes()[..32], &user_authority_delegate.to_bytes()[..32]],
         bump = delegate_bump
     )]
-    pub user_authority_delegate_pda: Account<'info, UserAuthorityDelegate>,
+    pub new_user_authority_delegate: Account<'info, UserAuthorityDelegate>,
+    /// CHECK: Signer/user delegate, can be defaulted to SystemProgram for no-op
     #[account()]
-    pub user_authority: Signer<'info>,
+    pub signer_user_authority_delegate: AccountInfo<'info>, // provided for authority authentication
```

Comment:
both instances of `signer_user_authority_delegate` could use a bit more detail here i think? as in this is the optional UserAuthorityDelegate account that is signing in turn for a user or something (will leave exact wording to you)